scripts/agentic/reconcile_devin_wiki_access.py

```
@@ -0,0 +1,340 @@
+@dataclass
+class Finding:
+    repository: str
+    state: str
+    operation: str
+    detail: str
+    installation_id: int | None = None
+
+
+def _validate_repository(full_name: str) -> str:
+    if not SAFE_REPOSITORY.fullmatch(full_name):
+        raise ReconcilerError(f"unsafe repository name returned by GitHub: {full_name!r}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)